tests/server/test_server.py

```diff
@@ -654,8 +680,24 @@ async def test_inverse_predicate():
     )
 
     # Run
-    output = await sync_query(q, log_level='DEBUG')
-    assert_no_warnings_trapi(output)
+    output = await sync_query(q)
+
+    validate_message({
+        "knowledge_graph":
+            """
+            CHEBI:6801 biolink:treats MONDO:0005148
+            """,
+        "results": [
+            """
+            node_bindings:
+                n0 MONDO:0005148
+                n1 CHEBI:6801
+            edge_bindings:
+                n0n1 CHEBI:6801-MONDO:0005148
+            """
+        ]
+    },
```

Comment:
Whitespace is weird here.

Reply:
How do you think this whitespace should look? I think it's kinda ugly right now, but I'm not exactly sure how to make it better.